packages/data-stores/src/auth/controls.ts

```diff
@@ -68,5 +116,12 @@ export function createControls( clientCreds: WpcomClientCredentials ) {
 				},
 			} );
 		},
+		REMOTE_LOGIN_USER: ( { loginLinks }: RemoteLoginUserAction ) =>
+			Promise.all(
```

Comment:
I think that this should be awaited here!

Reply:
The `REMOTE_LOGIN_USER` function returns the promise (arrow function has no braces) so it does end up being awaited.

Reply:
Ahh my bad, brain fart :joy:

Reply:
![ezgif-6-75adb3e22b40](https://user-images.githubusercontent.com/1500769/76384144-00b73280-63c3-11ea-8e58-29297eae216d.gif)